docs/resources/User.md

```diff
@@ -58,7 +58,11 @@ There are other rules and restrictions not shared here for the sake of spam and
   "banner": "06c16474723fe537c283b8efa61a30c8",
   "accent_color": 16711680,
   "premium_type": 1,
-  "public_flags": 64
+  "public_flags": 64,
+  "avatar_decoration_data": {
+    "sku_id": "1144058844004233369",
```

Comment:
What is this sku_id used for?

Reply:
It identifies the actual decoration - sometimes they edit the image so there are different assets corresponding to the same decoration but the sku id will be the same for everyone. I think there is currently no way for bots to fetch additional information for an sku though.